WolfpackBot.Data/Models/Event.cs

```diff
@@ -24,7 +25,23 @@ public class Event
         public int? Round { get; set; }
         public string LastRoundDate { get; set; }
         public string LastRoundTrack { get; set; }
-        public ulong? StandingsMessageId { get; set; }
+        public string StandingsMessageIds { get; set; }
+        [NotMapped]
+        public ulong[] StandingsMessageUlongs
```

Comment:
I think this should probably be 

`ParseStandingsMessageIds()` 
and 
`SetStandingMessageIds(ulong[] ids)` methods.

just so its more obvious that other properties are being set in the background

Reply:
Fixed